engineyard/ey-cookbooks-dev-v7
# [FBZ-10516] Migrate Datadog Recipe to v7 for MySQL

Description of your patch
-------------
Migrate Datadog Recipe to v7 for MySQL

Recommended Release Notes
-------------
Migrate Datadog Recipe to v7 for MySQL

Estimated risk
-------------
Medium

Description of testing done
-------------
* Create a v7 environment with MySQL as DB
* Update the custom recipe with the details such as the API Key and Site to use
* Upload the custom recipe and click on *Apply* to trigger chef run
* Verify if datadog is running via `ps -ef | grep datadog`
* If possible, verify in datadog if the agent is properly reporting

QA Instructions
-------------
* Create a v7 environment with MySQL as DB
* Update the custom recipe with the details such as the API Key and Site to use
* Upload the custom recipe and click on *Apply* to trigger chef run
* Verify if datadog is running via `ps -ef | grep datadog`
* If possible, verify in datadog if the agent is properly reporting
